<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Article Management Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="AMTTableWidget" name="tableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="AMTSearchInput" name="searchInput" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="AMTPreviewWidget" name="previewWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewLibrary"/>
    <addaction name="actionOpenLibrary"/>
    <addaction name="actionSaveLibrary"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionDel"/>
    <addaction name="actionEdit"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="actionUpdate"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuRemote">
    <property name="title">
     <string>&amp;Remote</string>
    </property>
    <addaction name="actionArxiv"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRemote"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionDel"/>
   <addaction name="actionEdit"/>
   <addaction name="actionSearch"/>
   <addaction name="actionPreview"/>
   <addaction name="actionDebug"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Open about window</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add entry</string>
   </property>
   <property name="toolTip">
    <string>Add entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="text">
    <string>Remove entry</string>
   </property>
   <property name="toolTip">
    <string>Delete entry</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Refresh table</string>
   </property>
   <property name="toolTip">
    <string>Retrieve entries from DB</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionNewLibrary">
   <property name="text">
    <string>New lib</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenLibrary">
   <property name="text">
    <string>Open lib</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveLibrary">
   <property name="text">
    <string>Save lib</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save lib as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit entry</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
   <property name="toolTip">
    <string>Show search bar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="text">
    <string>Preview</string>
   </property>
   <property name="toolTip">
    <string>Preview entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPullById">
   <property name="text">
    <string>Pull by id</string>
   </property>
  </action>
  <action name="actionArxiv">
   <property name="text">
    <string>arxiv.org</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AMTTableWidget</class>
   <extends>QTableView</extends>
   <header>amt/views/customWidgets/amttable</header>
  </customwidget>
  <customwidget>
   <class>AMTSearchInput</class>
   <extends>QWidget</extends>
   <header>amt/views/customWidgets/amtinputwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AMTPreviewWidget</class>
   <extends>QWidget</extends>
   <header>amt/views/customWidgets/amtpreview</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
